backend/database/migrations/001_khouryclassroom.sql

```diff
@@ -24,19 +16,30 @@ END $$;
 
 CREATE TABLE IF NOT EXISTS users (
     id SERIAL PRIMARY KEY,
-    first_name VARCHAR(255) NOT NULL,
-    last_name VARCHAR(255) NOT NULL,
+    first_name VARCHAR(255), --TODO: this should be not null eventually
+    last_name VARCHAR(255), --TODO: this should be not null eventually
     github_username VARCHAR(255) NOT NULL, 
-    github_user_id INTEGER NOT NULL,
-    role USER_ROLE NOT NULL
+    github_user_id INTEGER NOT NULL
```

Comment:
this should have a created_at field for logging and such

Reply:
I put in a ticket for this, don't think it's relevant to this PR